README.md

```diff
@@ -186,8 +186,11 @@ If not using vscode and the Dev Container, one can setup an environment manually
 A Linux environment assumed (WSL on Windows untested but probably supported). The following applications must be installed in order to instantiate the dev environment.
 
 - docker
-- docker-compose
-- node js
+- docker compose plugin v2
+- node js, including npm
+- clang
+- jq
+- psql
```

Comment:
Thanks for updating the list!

AFAIK all `psql` calls should be from within the postgres docker container so psql isn't required for the dev environment.

Reply:
I suspect that the psql error originates from within the postgres container and the error may have been related to something else. ... but there's no harm in over recommending software to install for a dev environment 😄 

Thanks again.